js/.eslintrc.cjs

```diff
@@ -1,113 +0,0 @@
-/** @type {import("eslint").Linter.Config } */
-module.exports = {
-  root: true,
-  ignorePatterns: [
-    "node_modules",
-    "dist",
-    "target",
-    "keycloak-theme",
-    "server",
-    // Keycloak JS follows a completely different and outdated style, so we'll exclude it for now.
-    // TODO: Eventually align the code-style for Keycloak JS.
-    "libs/keycloak-js",
-  ],
-  parserOptions: {
-    tsconfigRootDir: __dirname,
-    project: "./tsconfig.eslint.json",
-    extraFileExtensions: [".mjs"],
-  },
-  env: {
-    node: true,
-  },
-  plugins: ["lodash"],
-  extends: [
-    "eslint:recommended",
-    "plugin:import/recommended",
-    "plugin:import/typescript",
```

Comment:
I had to remove the import plugin due to incompatibility with the new 'flat' format (see https://github.com/import-js/eslint-plugin-import/issues/2556). Thankfully, we're not really using this plugin, so we can re-add it later when support eventually lands.